### ISSUE TITLE
Solid Forum license

### ISSUE THREAD

**Issue by csarven** [solid/solidproject.org#864]
Solid Forum indicates the following license in its ToS https://forum.solidproject.org/tos#3 :

>User contributions are licensed under a [Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License](http://creativecommons.org/licenses/by-nc-sa/3.0/deed.en_US).

Earliest archived snapshot at https://web.archive.org/web/20190221091610/https://forum.solidproject.org/tos#3

Besides the point of this issue but: I've raised concerns about this license in the past (going back to Gitter days and random meetings). I won't be able to repeat/recall exactly - unless people really want to dig into archives - but the CC NC-SA 3.0 license on the Forum never particularly matched the kind of licensing we've been using in the Solid project.

If anyhting from a Creative Commons license that should be used that's in some way aligned with the MIT license, it is probably CC BY 4.0: https://creativecommons.org/licenses/by/4.0/

So, I hope we can reconsider the license to be better aligned with the project.

**Comment by csarven** [comment]
See also https://solidproject.org/license :

>[Solid software](https://github.com/solid) is co-licensed under the MIT license and the Creative Commons Attribution 4.0 International (CC BY 4.0) license to cover both code and copy respectively. The individual licensors of each repository are mentioned in both the license.md of each repo as well as in the overview table below.

>The Solid Logo is under the CC-BY 4.0 license and was designed by Guillaume Baudusseau.

**Comment by TallTed** [comment]
Hmmm...

['Can You Use “Both” and “As Well As” Together? No, you cannot.'](https://www.google.com/search?q=can+we+use+both+and+as+well+as+together)

`mentioned in both the license.md of each repo as well as in the overview table`
-> `mentioned in both the license.md of each repo and the overview table`
or -> `mentioned both in the license.md of each repo and in the overview table`

**Comment by melvincarvalho** [comment]
I believe it was raised here:

https://github.com/solid/solid/issues/174

**Comment by michielbdejong** [comment]
So IIUC the proposal is to change the forum contributions license from CC NC-SA 3.0 to CC-BY 4.0 right?

I created a forum discussion about it: https://forum.solidproject.org/t/change-license-of-contributions-on-this-forum/7535
If people feel strongly that a change of license is worth the work this would take, then the Solid Team could pick this up.

Closing this here since IIUC it's unrelated to this repository.
